python/graphframes/tests.py

```diff
@@ -227,6 +261,18 @@ def test_page_rank(self):
         pr = g.pageRank(resetProb, tol=errorTol)
         self._hasCols(pr, vcols=['id', 'pagerank'], ecols=['src', 'dst', 'weight'])
 
+    def test_parallel_personalized_page_rank(self):
+        if GraphFrameTestUtils.is_later_version("2.1"):
```

Comment:
is it better to do `@unittest.skipIf` https://docs.python.org/2/library/unittest.html
this way it's indicated the test is skipped?